packages/react-core/src/components/Spinner/Spinner.tsx

```diff
@@ -16,24 +16,42 @@ export interface SpinnerProps extends Omit<React.HTMLProps<HTMLSpanElement>, 'si
   size?: 'sm' | 'md' | 'lg' | 'xl';
   /** Aria value text */
   'aria-valuetext'?: string;
+  /** Whether to use an SVG (new) rather than a span (old) */
+  isSVG?: boolean;
+  /** Diameter to set for SVG spinner */
+  diameter?: string;
 }
 
 export const Spinner: React.FunctionComponent<SpinnerProps> = ({
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   className = '',
   size = 'xl',
   'aria-valuetext': ariaValueText = 'Loading...',
+  isSVG = false,
+  diameter,
   ...props
-}: SpinnerProps) => (
-  <span
-    className={css(styles.spinner, styles.modifiers[size], className)}
-    role="progressbar"
-    aria-valuetext={ariaValueText}
-    {...props}
-  >
-    <span className={css(styles.spinnerClipper)} />
-    <span className={css(styles.spinnerLeadBall)} />
-    <span className={css(styles.spinnerTailBall)} />
-  </span>
-);
+}: SpinnerProps) => {
+  const Component = isSVG ? 'svg' : ('span' as any);
+
+  return (
+    <Component
+      className={css(styles.spinner, styles.modifiers[size], className)}
+      role="progressbar"
+      aria-valuetext={ariaValueText}
+      {...(isSVG && { viewBox: '0 0 100 100' })}
+      {...(isSVG && diameter && { style: { '--pf-c-spinner--diameter': diameter } })}
```

Comment:
`--pf-c-spinner--diameter` works with regular spinners, too. We just didn't have an example of it.

Reply:
Fixed!